### PR TITLE
feat: upload user avatar

### PR DESCRIPTION
**Pull request by johnytiago**

### CODE REVIEW

File: apps/asap-server/src/controllers/users.ts [hunk 1]
```diff
@@ -284,6 +290,28 @@ export default class Users {
     return parseGraphQLUser(items[0]);
   }
 
+  async updateAvatar(id: string, avatar: Buffer): Promise<UserResponse> {
+    const form = new FormData();
+    form.append('file', avatar, {
+      filename: `${id}.jpg`,
+      contentType: 'image/jpeg',
```
Comment (kmaid) [inline apps/asap-server/src/controllers/users.ts:297]: Are we enforcing only jpeg? I have allowed image/* in the frontend

File: apps/asap-server/test/handlers/users/update-avatar.test.ts [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+import { handler } from '../../../src/handlers/users/update-avatar';
+import { apiGatewayEvent } from '../../helpers/events';
+import { identity } from '../../helpers/squidex';
+import { body } from './update-avatar.fixtures';
+import { patchResponse } from './update.fixtures';
+import decodeToken from '../../../src/utils/validate-token';
+
+jest.mock('../../../src/utils/validate-token');
+
+describe('PATCH /users/{id}/avatar - validations', () => {
```
Comment (kmaid) [inline apps/asap-server/test/handlers/users/update-avatar.test.ts:14]: In the frontend we are using post as you have required in the handler

**Review by kmaid — CHANGES_REQUESTED** [review]

File: apps/asap-server/test/handlers/users/update-avatar.test.ts (hunk 2 above)
Comment (johnytiago) [reply apps/asap-server/test/handlers/users/update-avatar.test.ts:14]: Yeah, this is just a copy & paste error. We don't look into the request method during tests.

File: apps/asap-server/src/controllers/users.ts (hunk 1 above)
Comment (johnytiago) [reply apps/asap-server/src/controllers/users.ts:297]: Can you pass the content type as an header? I don't want to be looking into the buffer to figure that out 😅

Comment (johnytiago) [reply apps/asap-server/src/controllers/users.ts:297]: As discussed on zoom: the backend will implement a dataURI parser and extract the type from that.

**Pull request merged**